css/home.css

```diff
@@ -2,217 +1,170 @@
+.dropdown>.dropdown-toggle:active {
+  /*Without this, clicking will make it sticky*/
+    pointer-events: none;
 }
 
-.kata-sambutan {
-  display: inline;
-  padding: 45px;
+.bg-white{
+  background-color: white;
 }
 
-#sambutan {
-  display: inline-block;
-  color: #fff;
-  background-color: #eaba3a;
-  padding: 8px;
-  border-radius: 2px;
+.bgCarousel {
+  height: 100vh;
+  min-height: 350px;
+  background: no-repeat center center scroll;
+  background-attachment: fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  z-index: -2;
 }
 
-article {
-  display: grid;
-  grid-template-columns: 1fr 1fr;
-  grid-gap: 10px;
+.bgCarousel:hover {
+  cursor: w-resize;
 }
 
-.artitle {
-  grid-column: 1 / 3;
-  font-size: xx-large;
-  font-weight: bold;
-  color: #03153e;
+.parallax-text {
+  position: fixed;
+  color: white;
+  z-index: -1;
+  top: 150px;
+  left: 100px;
+  width: 500px;
 }
 
-.list_berita {
-  display: grid;
-  grid-template-columns: auto;
-  grid-gap: 10px;
+.custom-btn {
+  text-decoration: none;
+  padding: 15px;
+  color: white;
+  display: inline-block;
 }
 
-.berita {
-  display: grid;
-  grid-template-columns: 1fr 3fr;
-  grid-template-rows: 1fr 4fr;
-  grid-gap: 10px;
+/* .swiper-container {
+  width: 100%;
+  height: 100%;
 }
 
-.berita .b_gambar {
-  grid-row: 1 / 3;
+.swiper-slide {
+  height: 100vh;
+  min-height: 350px;
+  background: no-repeat;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
 }
 
-.berita .b_gambar img {
-  height: 100px;
-  width: 180px;
-  object-fit: cover;
-}
+.swiper-slide:hover {
+  cursor: w-resize;
+} */
 
-.berita .b_isi {
-  align-self: start;
+.flyText {
+  background-color: rgba(255, 255, 255, 0.514);
+  width: 400px;
+  padding: 20px;
+  right: 150px;
+  bottom: 80px;
+  position: absolute;
 }
 
-/* Footer */
-footer {
-  display: grid;
-  grid-template-rows: auto auto;
-  grid-gap: 20px;
-  background-color: #03153e;
-  color: white;
+.text-primary {
+  text-align: center;
+  margin-top: 30px;
+  margin-bottom: 20px;
 }
 
-.fcolor1 {
-  font-weight: bold;
-  font-size: medium;
+.ceo {
+  padding: 45px 0;
 }
 
-footer a,
-.fcolor2 {
-  text-decoration: none;
-  color: #b7b7b7;
+#sambutan {
+  background-color: #eaba3a;
+  display: inline-block;
+  color: #fff;
+  font-size: 18px;
+  padding: 10px;
 }
 
 .footer1 {
-  display: grid;
-  grid-template-columns: 2fr 1fr 1fr 1fr;
-  grid-gap: 10px;
-  padding: 30px;
-}
-
-.footer1 .profil {
-  display: grid;
-  grid-template-columns: auto;
-}
-
-.footer1 .profil div img {
-  width: 180px;
-}
-
-.footer1 div {
-  display: grid;
-  grid-template-rows: 1fr auto;
-  grid-gap: 10px;
-  align-self: start;
+  background-color: #03153e;
 }
 
-.footer1 div .link a img {
-  width: 220px;
+.footer2 {
+  background-color: #020e28;
 }
 
-.sosmed {
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr 1fr;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".grayLink a" to come before selector ".dropdown-menu a:hover" (no-descending-specificity)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)